<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPlayerMiniature</class>
 <widget class="QWidget" name="FormPlayerMiniature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>167</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelAvatar">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNick">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font-weight: 700;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="ProgressBarHp" name="progressBarHp">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%v/%m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ProgressBarMp" name="progressBarMp">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%v/%m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ProgressBarExp" name="progressBarExp">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonKick">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressBarHp</class>
   <extends>QProgressBar</extends>
   <header>class/widgets/progressbarhp.h</header>
  </customwidget>
  <customwidget>
   <class>ProgressBarMp</class>
   <extends>QProgressBar</extends>
   <header>class/widgets/progressbarmp.h</header>
  </customwidget>
  <customwidget>
   <class>ProgressBarExp</class>
   <extends>QProgressBar</extends>
   <header>class/widgets/progressbarexp.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
